internal/security/config/command/proxy/command.go

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2020 Intel Corporation
+// Copyright (c) 2020-2023 Intel Corporation
 //
 // SPDX-License-Identifier: Apache-2.0'
```

Comment:
sorry, just saw this:
```suggestion
// SPDX-License-Identifier: Apache-2.0
```

Reply:
fixed in rebase